tools/tf2openapi/types/tfsignaturedef_test.go

```diff
@@ -9,6 +9,15 @@ import (
 	"testing"
 )
 
+/** Utils **/
+func newBool(b bool) *bool {
```

Comment:
Consider just Bool to match your other type. Should we extract these helpers somewhere else? I'd like to use them myself in KFServing's controller :)

Reply:
Got it. extracting to pkg/utils/utils.go